### PR TITLE
yotam's edit

### PR DESCRIPTION
**Pull request by yotam-stsci**

### CODE REVIEW

**Comment by nkerman** [comment]
Hi Yotam! Where's the Hello world?????? 😄

**Comment by yotam-stsci** [comment]
Oh shoot.... uhh.. Hello World!!!

**Comment by nkerman** [comment]
I think you want to first update your new file to say just "Hello World" but whatever for training's sake, I'm just trying to write this like a real PR review.

File: trainees/hello_150.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+yotam was here
```
Comment (nkerman) [inline trainees/hello_150.txt:1]: Should be 'Hello world!'

**Comment by bourque** [comment]
I am labeling this as INVALID 😛

**Comment by bourque** [comment]
On second thought this is a nice enhancement

**Comment by yotam-stsci** [comment]
Thank you @bourque, I'm glad you see the brilliance of my ways

**Pull request merged**